settings.py

```diff
@@ -376,6 +376,10 @@
 
 SERVICE_ACCOUNT_FOR_ANVIL = None
 
+AIRFLOW_CLIENT_ID = os.environ.get('AIRFLOW_CLIENT_ID')
+AIRFLOW_WEBSERVER_ID = os.environ.get('AIRFLOW_WEBSERVER_ID')
+DAG_VERSION = '0.0.1'
```

Comment:
`DAG_VERSION` should be in `anvil_workspace_api.py`, it doesn't need to be a shared setting